service_configuration_lib/spark_config.py

```diff
@@ -291,31 +291,38 @@ def get_dra_configs(spark_opts: Dict[str, str]) -> Dict[str, str]:
     ):
         return spark_opts
 
+    spark_app_name = spark_opts.get('spark.app.name', '')
+
     log.warning(
-        'Spark Dynamic Resource Allocation (DRA) enabled for this batch. More info: y/spark-dra. '
-        'If your job is performing worse because of DRA, consider disabling DRA. To disable, '
-        'please provide spark.dynamicAllocation.enabled=false in your spark args\n',
+        'Spark Dynamic Resource Allocation (DRA) enabled for this batch. More info: y/spark-dra.\n',
     )
+    # TODO: add regex to better match Jupyterhub Spark session app name
+    if 'jupyterhub' not in spark_app_name:
+        log.warning(
+            'If your job is performing worse because of DRA, consider disabling DRA. To disable, '
```

Comment:
Now that we have a successful mass rollout, easily available documentation, I don't think this message is needed anymore.

Reply:
Removed